lib/gollum-lib/filter/tags.rb

```diff
@@ -357,4 +358,8 @@ def generate_image_attributes(options)
 
     return classes, attrs, containered
   end
+  
+  def query_for_path(path)
+    @markup.wiki.hyphened_tag_lookup ? path.gsub(' ', '-') : path
```

Comment:
Just to be clear: we don't want a fallback construction here, correct? That if no page/file is found with spaces that we look for a hyphenated version. Seems like that would be a lot of overhead, but just checking.

Reply:
Yes, exactly. And it would raise the question of how to combine that fallback with the `global_tag_lookup`: one fallback search that enables both `global_tag_lookup` and `hyphened_tag_lookup`? Two fallback searches? Etc. So think it's best to just enable this from the very first search on. Means that with `--hyphened-tag-lookup` people cannot use spaces in filenames at all -- but that seems to be the ask.

Reply:
OK!